cv-basics/Playing_with_matrices/Playing_with_matrices.ipynb

```diff
@@ -0,0 +1,857 @@
+{
```

Comment:
<p>You can print time used for calculating to show the efficiency difference between naive approach and numpy functions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SRA-VJTI/Intro_to_CV/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>